kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/ProcessServicesClient.java

```diff
@@ -93,5 +94,11 @@
 
     List<WorkItemInstance> getWorkItemByProcessInstance(String containerId, Long processInstanceId);
 
+    List<NodeInstance> findActiveNodeInstances(String containerId, Long processInstanceId, Integer page, Integer pageSize);
```

Comment:
Duplicated from QueryServicesClient so user don't have to switch clients to get info?